client/java/src/main/java/com/linecorp/centraldogma/client/AbstractWatcher.java

```diff
@@ -43,22 +43,31 @@
 import com.google.common.base.MoreObjects;
 
 import com.linecorp.centraldogma.common.CentralDogmaException;
+import com.linecorp.centraldogma.common.Commit;
 import com.linecorp.centraldogma.common.EntryNotFoundException;
 import com.linecorp.centraldogma.common.PathPattern;
 import com.linecorp.centraldogma.common.Query;
 import com.linecorp.centraldogma.common.RepositoryNotFoundException;
 import com.linecorp.centraldogma.common.Revision;
 
+import io.micrometer.core.instrument.Meter.Id;
+import io.micrometer.core.instrument.Meter.Type;
+import io.micrometer.core.instrument.MeterRegistry;
+import io.micrometer.core.instrument.Tags;
+
 abstract class AbstractWatcher<T> implements Watcher<T> {
 
     private static final Logger logger = LoggerFactory.getLogger(AbstractWatcher.class);
+    private static final String LATEST_REVISION_METER_NAME = "centraldogma.watcher.latest.revision";
+    private static final String LATEST_COMMIT_TIME_METER_NAME = "centraldogma.watcher.latest.commit.time";
```

Comment:
```suggestion
    private static final String LATEST_REVISION_METER_NAME = "centraldogma.client.watcher.latest.revision";
    private static final String LATEST_COMMIT_TIME_METER_NAME = "centraldogma.client.watcher.latest.commit.time";
```

Some people may want to use a different prefix and add custom tags for their metrics. 
What do you think of adding a customization point for that? 

I realized that the `centraldogma-client` module has no dependency on Armeria. Actually, we have decided to add Armeria dependency to `centraldogma-client` but didn't put it in action.

To conclude, let's add Armeria dependency and use `MeterIdPrefix`, which I proposed in the issue.

Reply:
I also wrote this in the README, but since this is a relatively unique metric, I couldn’t think of many use cases for customization at this point.

I considered using this, but copying and pasting it didn’t seem ideal, so I decided not to for now.
(That said, it's also not a good idea for centraldogma-client to depend on Armeria.)
[MeterIdPrefix in Armeria](https://github.com/line/armeria/blob/main/core/src/main/java/com/linecorp/armeria/common/metric/MeterIdPrefix.java)

And if we want to rename or retag metrics, we can transform them using a meter filter.
[Micrometer Meter Filters Documentation](https://github.com/micrometer-metrics/micrometer-docs/blob/main/src/docs/concepts/meter-filters.adoc#transforming-metrics)

...How about?

Reply:
I see. How about just adding an infix `.client`? I want to distinguish it from the server metric. We may add additional metrics under `centraldogma.client` namespace.

Reply:
Ah, I overlooked the client infix remark. You're absolutely right. I'll fix it!